apps/web-mzima-client/src/app/feed/feed.component.ts

```diff
@@ -876,11 +884,8 @@ export class FeedComponent extends MainViewComponent implements OnInit, OnDestro
     }
   }
 
-  refreshPost({ id }: PostResult) {
-    this.postsService.getById(id).subscribe((p) => {
-      const updatedPost = _.cloneDeep(p);
-      this.posts = this.posts.map((obj) => (obj.id === updatedPost.id ? updatedPost : obj));
-    });
+  refreshPost() {
+    this.getPosts({ params: this.params });
```

Comment:
Instead, all you need to do is to use task of `startCount` to control scroll back to number `1` after the refresh (example in line `334` window resize function).

Reply:
What I mean is to call the scroll function/method under this line `this.getPosts({ params: this.params });` in the reset function block, and then using `startCount` task here.
Instead of changing the task for the one in the router navigation end @Angamanga